src/AbstractPathMappingRepository.php

```diff
@@ -330,4 +330,48 @@ protected function resolveRelativePaths($relativePaths)
 
         return $relativePaths;
     }
+
+    /**
+     * Check if the given target path represents a link.
+     *
+     * @param string $targetPath
+     *
+     * @return bool
+     */
+    public static function isLinkTarget($targetPath)
```

Comment:
Not really, and not even static. I initially used them in the tests but not anymore. I'll set them as private.